Clarifications on ZIP 271: Deferred Dev Fund Lockbox Disbursement (#1071)
zcash/zips: zips/zip-0271.md

```

> where NU6MainnetActivationHeight is 2726400 [^zip-0253] and 0.1875 ZEC is the

Reviewer: I checked this and the formula above.

## Extension to the lockbox funding stream

The ``FS_DEFERRED`` lockbox funding stream is set to receive 12% of the block subsidy and
is extended by 1,260,000 blocks up to a maximum height of 4406400  on both Mainnet and Testnet.

> If this ZIP activates before or on the first block after the end of current dev fund as of 

Author: ```suggestion
This ZIP activates before or on the first block after the end of current dev fund as of 
```

> The ``FS_DEFERRED`` lockbox funding stream is set to receive 12% of the block subsidy and
> is extended by 1,260,000 blocks up to a maximum height of 4406400  on both Mainnet and Testnet.
> This ZIP activates before or on the first block after the end of current dev fund as of 
> this writing and specified in [^zip-1015-funding-streams], this would extend ``FS_DEFERRED`` on
> Mainnet until block height 4406400.

Author: ```suggestion
The ``FS_DEFERRED`` lockbox funding stream is set to receive 12% of the block subsidy and
is extended by 1,260,000 blocks on both Mainnet and Testnet.
This ZIP activates before or on the first block after the end of current dev fund as of 
this writing and specified in [^zip-1015-funding-streams], and will extend ``FS_DEFERRED`` on
Mainnet until block height 4406400.
```
